plugins/widget/plugin.js

```diff
@@ -452,6 +452,10 @@
 				if ( !element )
 					return null;
 
+				if ( !( element instanceof CKEDITOR.dom.element ) ) {
```

Comment:
The fix is ok, but we may take it one step further. The root cause of the issue is `mousedown` listener which checks if widget was clicked (https://github.com/ckeditor/ckeditor-dev/blob/master/plugins/widget/plugin.js?utf8=%E2%9C%93#L2503). You may just check there if event target is document type and stop the execution if it is (btw. there is already one similar workaround for IE8 there - https://github.com/ckeditor/ckeditor-dev/blob/master/plugins/widget/plugin.js?utf8=%E2%9C%93#L2507).